docs/units/physical_types.rst

```diff
@@ -0,0 +1,56 @@
+.. |PhysicalType| replace:: :class:`~astropy.units.PhysicalType`
+.. |Quantity| replace:: :class:`~astropy.units.Quantity`
```

Comment:
With #11599 in imminently, |Quantity| is already taken care of. |PhysicalType| might be a good addition to conf.py as well

Reply:
#11599 is now merged - agreed that `PhysicalType` might as well be in `conf.py` as well.